NotificationHubs/src/com/windowsazure/messaging/RuntimeTypeAdapterFactory.java

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (C) 2011 Google Inc.
```

Comment:
Woah - I've never shipped code under a different license belonging to a different company in a Microsoft open source project before. Let's sync up and see what company guidance is applicable here.

Reply:
I'm going to remove it because it's not applicable anymore as we're not using it for polymorphic deserialization.